fili-core/src/test/groovy/com/yahoo/bard/webservice/data/cache/MemcachedCacheSpec.groovy

```diff
@@ -160,7 +160,7 @@ class MemcachedCacheSpec extends Specification {
             .request().get(String.class)
 
         then: "should read old result from cache"
-        GroovyTestUtils.compareJson(result, expected)
+        GroovyTestUtils.compareJson(result, '{"rows":[]}')
```

Comment:
Per my comment above about removal vs. deprecation, this behavior shouldn't be changing yet.